### ISSUE TITLE
"does not contain a definition for 'Text'" in vscode

### ISSUE THREAD

**Issue by xxSeys1** [FlaxEngine/ImGui#4]
I get this error in Vscode (vscode only, not in vs):

![image](https://github.com/user-attachments/assets/d176d6e6-5458-49a2-af23-225ae46f6062)

I'm not sure if that's an issue with this addon itself or if it's a vscode issue, but I thought that maybe someone knows a solution here to get rid of that pesky error.
The error only shows in vscode, nothing in the Flax output log. Also the code runs just fine when I enter play mode.

I have the C# vscode extension installed and also tried re- generating my project files (which did not help).

This could be related to #3 , but I'm not sure about that so I'm opening a new issue.

**Comment by mafiesto4** [comment]
I've got it working with the latest `master` (as of today):

![Image](https://github.com/user-attachments/assets/1412debb-8588-4d7c-a080-baf25fa66268)

**Comment by xxSeys1** [comment]
I get autocomplete n stuff now too, nice :)
But I still don't get any autocomplete for the `ImGui` class, only after I change it from
```cs
partial class ImGui
```

to

```cs
public partial class ImGui
```

in *ImGui.cs* L 1246

*Edit*
And there also is no autocomplete for
```cs
ImGui.Text()
```
![image](https://github.com/user-attachments/assets/d22a1f13-8bdb-474a-afa9-264809fb5713)

**Comment by mafiesto4** [comment]
I added this suggestion in 019279ced830db44a92f7014ab1e59e0bd77b0db. After testing again it works as intended on the latest `master` (engine version).
